### ISSUE TITLE
Widget.Grid gives error msg

### ISSUE THREAD

**Issue by TanR314** [linkfrg/ignis#75]
### Already reported ? *

- [x] I have searched the existing open and closed issues.

### System Information

 ```
Ignis 0.3.dev0
Branch: 
Commit:  ()
Current desktop: Hyprland

os-release:
ANSI_COLOR="0;38;2;126;186;228"
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
BUILD_ID="25.05.20241211.5d67ea6"
CPE_NAME="cpe:/o:nixos:nixos:25.05"
DEFAULT_HOSTNAME=nixos
DOCUMENTATION_URL="https://nixos.org/learn.html"
HOME_URL="https://nixos.org/"
ID=nixos
ID_LIKE=""
IMAGE_ID=""
IMAGE_VERSION=""
LOGO="nix-snowflake"
NAME=NixOS
PRETTY_NAME="NixOS 25.05 (Warbler)"
SUPPORT_URL="https://nixos.org/community.html"
VARIANT=""
VARIANT_ID=""
VENDOR_NAME=NixOS
VENDOR_URL="https://nixos.org/"
VERSION="25.05 (Warbler)"
VERSION_CODENAME=warbler
VERSION_ID="25.05"
```

### Description

```python
Widget.Grid(
  child=[Widget.Button(label=str(i)) for i in range(100)],
  row_num=3
)

```
This is not working and giving this: ( 100 times )
```
2024-12-15 10:32:27 [CRITICAL] (python3:6079): Gtk-CRITICAL **: 10:32:27.322: gtk_grid_attach: assertion '_gtk
_widget_get_parent (child) == NULL' failed
```

 

### How to reproduce

I guess the above code should reproduce it.

**Comment by newor0599** [comment]
Not working as in not able to see the 100 buttons?

**Comment by TanR314** [comment]
Yes

**Comment by linkfrg** [comment]
should be fixed after 1cd1460ca19b75d5efb4aa1a8b3b6350e5f40c44
